pkg/controller/apmserver/controller.go

```diff
@@ -276,6 +279,18 @@ func (r *ReconcileApmServer) doReconcile(ctx context.Context, request reconcile.
 	return res, err
 }
 
+func (r *ReconcileApmServer) warnIfDeprecated(version semver.Version, as *apmv1.ApmServer) {
+	if version.GTE(semver.MustParse("8.0.0")) {
+		message := "The standalone APM Server binary is deprecated as of version 8.0.0 and will be removed in a future release.  Consider using Elastic Agent in Fleet-managed mode."
```

Comment:
```suggestion
		message := "The standalone APM Server binary is deprecated as of version 8.0.0 and will be removed in a future release. Consider using Elastic Agent in Fleet-managed mode."
```

Reply:
argh, I'm trying to break this double space habit... 👍